src/bp/ui-admin/src/routes/index.tsx

```diff
@@ -53,9 +53,14 @@ export const makeMainRoutes = () => {
   setupBranding()
 
   const ExtractToken = () => {
-    const token = extractCookie('userToken')
-    if (token) {
-      authentication.setToken({ jwt: token })
+    const userToken = extractCookie('userToken')
+    const tokenExpiry = extractCookie('tokenExpiry')
+
+    if (userToken) {
```

Comment:
Maybe you should also check that `tokenExpiry` is defined
```suggestion
    if (userToken && tokenExpiry) {
```